packages/paste-website/src/pages/components/button-group/index.mdx

```diff
@@ -0,0 +1,252 @@
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(frontmatter: {slug: {eq: "/components/button-group/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "Button Group"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+          }
+        }
+      }
+    }
+  }
+`;
+
+<NormalizedComponentHeader
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/button-group"
+  storybookUrl="/?path=/story/components-button-group--attached"
+  data={props.data}
+  description={props.pageContext.frontmatter.description}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<LivePreview
+  noInline
+  scope={{
+    ButtonGroup,
+    Button,
+    PlusIcon,
+    CheckboxCheckIcon,
+    PopoverContainer,
+    PopoverButton,
+    Popover,
+    Heading,
+    Separator,
+    Text,
+  }}
+  language="jsx"
+>
+  {BasicExample}
+</LivePreview>
+
+## Guidelines
+
+A Button Group is a set of 2-6 buttons. It is often used to visually group related buttons together. The buttons should be independent from each other, i.e. clicking one button does not change the state of another button.
+
+Any Paste components that use [Button](/components/button), including [Popover](/components/popover), [Modal](/components/modal), and [Minimizable Dialog](/components/minimizable-dialog) can be put into a ButtonGroup.
+
+### Button Group vs Radio Button Group
+
+Radio Button Groups should be used when the user needs to pick one option from the group. Radio Buttons cannot be used wihtin a Button Group.
+
+## Examples
+
+### Default
+
+By default, the Buttons in a Button Group are displayed horizontally with a margin between them.
+
+<LivePreview
+  noInline
+  scope={{
+    ButtonGroup,
+    Button,
+    PlusIcon,
+    CheckboxCheckIcon,
+    PopoverContainer,
+    PopoverButton,
+    Popover,
+    Heading,
+    Separator,
+    Text,
+  }}
+  language="jsx"
+>
+  {UnattachedExample}
+</LivePreview>
+
+### Attached
+
+Use the `attached` prop to make the buttons visually connected.
+
+<LivePreview
+  noInline
+  scope={{
+    ButtonGroup,
+    Button,
+    PlusIcon,
+    CheckboxCheckIcon,
+    PopoverContainer,
+    PopoverButton,
+    Popover,
+    Heading,
+    Separator,
+    Text,
+  }}
+  language="jsx"
+>
+  {BasicExample}
+</LivePreview>
+
+### Icon
+
+Use the icon variant when space is limited and the icons being used are easily recognizable (e.g., icons representing graph types). It is highly recommended to pair this variant with the [Tooltip](/components/tooltip) component to help ensure understandability.
```

Comment:
For the example on the docs site, should we pair it with a tooltips since we highly recommend that?

Reply:
Totally! I'll update it.